docker/Dockerfile.amd

```diff
@@ -12,7 +12,10 @@ ENV rocm=/opt/rocm-${ROCM_VERSION}
 ENV AOMP=/opt/rocm-${ROCM_VERSION}/llvm
 
 # Some utils needed
-RUN apt-get update && apt-get install -y wget git autoconf dh-autoreconf flex python3-venv python3-dev vim
+RUN apt-get update && apt-get install -y wget git autoconf dh-autoreconf flex python3-venv python3-dev vim libnuma1 tmux
```

Comment:
This seem to have broken something. Des tmpi installs an MPI package as a dependency? That could be problematic